security-admin/src/test/java/org/apache/ranger/audit/TestAuditQueue.java

```diff
@@ -783,31 +783,49 @@ public void testAuditFileQueueSpoolORCRollover(){
 		String fileSpoolPrefix = filequeuePrefix + ".filespool";
 		props.put(fileSpoolPrefix+".dir",spoolFolderName);
 		props.put(fileSpoolPrefix+".buffer.size",""+100000000000000L);
-		props.put(fileSpoolPrefix+".file.rollover.sec",""+300);
+		props.put(fileSpoolPrefix+".file.rollover.sec",""+5);
 		AuditProviderFactory factory = new AuditProviderFactory();
 		factory.init(props, appType);
 		AuditHandler queue = factory.getAuditProvider();
-		for (int i = 0; i < messageToSend; i++) {
+		for (int i = 0; i < preRolloverMessagesCount; i++) {
 			queue.log(createEvent());
 			try {
-				Thread.sleep(5);
+				Thread.sleep(10);
 			} catch (InterruptedException e) {
 				System.out.println(e);
 			}
 		}
+		//wait for rollover to happen
 		try {
-			Thread.sleep(40000);
+			Thread.sleep(10000);
 		} catch (InterruptedException e) {
 			System.out.println(e);
 		}
+		//send some more logs
+		for (int i = 0; i < postRolloverMessagesCount; i++) {
+			queue.log(createEvent());
+			try {
+				Thread.sleep(10);
+			} catch (InterruptedException e) {
+				System.out.println(e);
```

Comment:
Please use logger.error.

Reply:
Fixed. Used logger instead of sysout at all places.